test/zipObj.js

```diff
@@ -11,9 +11,8 @@ describe('zipObj', function() {
     eq(R.zipObj(['a', 'b', 'c'], [1, 2, 3, 4, 5, 6, 7]), {a: 1, b: 2, c: 3});
   });
 
-  it('extra keys are undefined', function() {
```

Comment:
Sounds reasonable, thanks. I started digging through the history of the function and it looks like it's _always_ been as it is currently (pre-fix), for what that's worth.